test/02-investigate.t

```diff
@@ -178,6 +194,14 @@ test-post-investigate() {
     is "$rc" 0 'early return (other test in cluster) (30005)'
     has "$got" "is not a retry" "vim-other vs. vim (30005)"
 
+    # test issue
+    try investigate 30031
+    is "$rc" 2 'mocked function returned failure (30031)'
+    has "$got" "Commenting 3003" "Posting comment on OPENQA_INVESTIGATE_ORIGIN (30031)"
+    has "$got" "Investigate retry job **vim:investigate:retry**" "retry test name appears in comment(30031)"
+    has "$got" "likely not a sporadic" "not sporadic (30031)"
+    has "$got" "likely a test issue" "product issue (30031)"
```

Comment:
I assume the test description string should reference test issue, not product issue